tests/e2e/fine_grained_access/edge_type_filtering_tests.py

```diff
@@ -1,91 +1,116 @@
-import common
 import sys
+
+import common
 import pytest
 
 
-def test_all_edge_types_all_labels_granted():
+@pytest.mark.parametrize("switch", [False, True])
+def test_all_edge_types_all_labels_granted(switch):
     admin_connection = common.connect(username="admin", password="test")
-    user_connnection = common.connect(username="user", password="test")
+    user_connection = common.connect(username="user", password="test")
     common.execute_and_fetch_all(admin_connection.cursor(), "GRANT READ ON LABELS * TO user;")
     common.execute_and_fetch_all(admin_connection.cursor(), "GRANT READ ON EDGE_TYPES * TO user;")
 
-    results = common.execute_and_fetch_all(user_connnection.cursor(), "MATCH (n)-[r]->(m) RETURN n,r,m;")
+    if switch:
+        common.switch_db(user_connection.cursor())
```

Comment:
Are we sure that `clean` database exists? We don't call `common.create_multi_db(user_connection.cursor(), switch)` to create `clean` database.

Reply:
Oh I saw that we create database in `workloads.yaml`. Comment is resolved now.